### PR TITLE
Created a function to reset Account using firebase

### PR DESCRIPTION
**Pull request by OladipoOmotosho**
Created a function to reset Account using firebase

### CODE REVIEW

File: src/composables/resetAccountWithFirebase.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import { getAuth, updateEmail, updatePassword } from "firebase/auth"
```
Comment (annonymousauthority) [inline src/composables/resetAccountWithFirebase.js:1]: there's already an named export that is firebase configured, use that not `getAuth`

File: src/composables/resetAccountWithFirebase.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+function resetAccount(email, newPassword) {
+    return new Promise((resolve, reject) => {
+        const auth = getAuth();
+        const user = auth.currentUser;
```
Comment (annonymousauthority) [inline src/composables/resetAccountWithFirebase.js:6]: The initial comment should reflect here

File: src/composables/resetAccountWithFirebase.js
Comment (annonymousauthority) [inline src/composables/resetAccountWithFirebase.js:1]: This entire function doesn't solve the issue that was opened.

Could you address this

**Review by annonymousauthority — CHANGES_REQUESTED** [review]

**Comment by OladipoOmotosho** [comment]
Function written did not address the issue opened

**Pull request closed without merging**